client/components/marketing-survey/gsuite-cancel-purchase-dialog/style.scss

```diff
@@ -0,0 +1,14 @@
+.gsuite-cancel-purchase-dialog__dialog {
+	max-width: 550px;
+}
+
+// adds a small amount of space to the bottom of the features to prevent
+// it from overlapping with the "fade" at the bottom of the dialog at narrow widths
+.gsuite-cancel-purchase-dialog__features::after {
+	content: '';
+	display: block;
+	height: 1em;
+	@include breakpoint( '>480px' ) {
+		content: none;
+	}
+}
```

Comment:
We could attack this from the opposite direction, which gives a slightly simpler code:

```css
@include breakpoint( '<480px' ) {
	.gsuite-cancel-purchase-dialog__features::after {
		content: '';
		display: block;
		height: 1em;
	}
}
```

Reply:
I prefer the mobile first approach.  But am open to either.